### ISSUE TITLE
[Web] Please allow to "unselect" any active chat pressing ESC like in Telegram

### ISSUE THREAD

**Issue by Polve** [wireapp/wire-webapp#7022]
Ditto.

It's very handy to be able to avoid showing the chat page to someone passing by just by pressing ESC like in telegram desktop.

I'm unable to find a way to do so in wire, I must minimize the window or select another chat window.

**Comment by bennycode** [comment]
Interesting feature request but we don't plan to do this. If you are curious about people watching your back you should consider getting a privacy filter for your physical screen.

**Comment by Polve** [comment]
In telegram every user is able to use this simple feature without the need of a "physical privacy filter for the screen".

I can't understand why to close such a basic feature request without even a rationale.

**Comment by maicardi** [comment]
Well you can even lock your workstation with Win + L or Ctrl + Alt + L, or you can even cut the power off..

Anyway hiding the current chat with Esc is a painless addition to make which could came handy to many users.

**Comment by bennycode** [comment]
@Polve & @maicardi if you find it very important to unselect any active chat by pressing 'Esc' like in Telegram you can submit a pull request to our code and we will review it.
